Nodejs/Product/Nodejs/SharedProject/CommonFolderNode.cs

```diff
@@ -227,14 +227,14 @@ internal override int IncludeInProjectWithProgress(bool includeChildren)
         public override void RenameFolder(string newName)
         {
             var oldName = this.Url;
-            this._project.SuppressFileChangeNotifications();
+//            this._project.SuppressFileChangeNotifications();
```

Comment:
Delete?

Reply:
There seems to be a bunch of commented-out code.

---
In reply to: [160538343](https://github.com/Microsoft/nodejstools/pull/1835#discussion_r160538343) [](ancestors = 160538343)